README.md

```diff
@@ -1,2 +1,57 @@
+ 3. Abdalla, A., Alashwal, H., Crouse, J., El Halaby, M., & Moustafa, A. (2019). The Application of Unsupervised Clustering Methods to Alzheimer’s Disease. Frontiers in Computational Neuroscience, 13, 31. https://dx.doi.org/10.3389%2Ffncom.2019.00031 
+
+Each journal above investigates a portion of which we wish to acheive:
+1. Bayesian networks readily resolve our foreseen issue regarding the different chances that some set of symptoms is related to a particular illness: we'll be able to exploit the probabilistic model the algorithm represents and report the chances of a particular illness.
+
+The article by Auras, et. al (2016) presents the use of a Bayesian Network in the diagnosis of breast-related diseases using legitimate medical data. However, we do not have the same access to legitimate medical data and as such need to investigate an alternative approach to training a Bayesian Network.
+
+2. Similarly to Bayesian networks, an ANN provide us with a model that connects symptoms to illnesses. Additionally, the strucutre of ANN allows us to correct probabilities by comparing the ANN computed illness to the actual illness confirmed by the user and then adjusting the ANN structure.
+
+The article by Govrdhan, et. al (2010) presents the use of an ANN when mining medical data for the prediction of heart attacks. The system is first built using data and then is training on new data as it arrives. Again, as we do not have access to legitimate medical data our ANN would be training on arriving data only.
+
+3. Similar to the algorithms above, k-means clustering provides us with a model that can be used to predict input-to-output relationships. Interestingly, the model is not based on probability statistics and instead is used to find natural groupings between inputs as the algorithm executes. We could compare the clusters of symptoms to known sets of symptoms-to-illness from reputable clinical websites and report.
+
+The article by Abdalla, et. al (2019) investigates the use of k-means clusters to determine the symptoms of Alzheimer’s disease to better predict that a group of symptoms is predictive of the disease. K-means clustering, like Bayesian networks, usually operates from an initial set of data - however, as the k-means clustering does not need the probabilities of symptoms to illness we could use legitimate sypmtoms-to-illness relationship data as our first training set.
+
+As we, the team are not medical professionals we do not have accurate and legitimate symptoms-to-illness relationship data at our disposal. We intend to retrieve legitimate data from reputable clinical websites such as [Mayo Clinic](www.mayoclinic.org) (under the clause that we do not use the data for commercial use) using web scraping aided by respective Python libraries and modules.
```

Comment:
I think we've moved away from needed web-scraping to get data off the internet because sites like [Kaggle](https://www.kaggle.com/) exists.

Reply:
Updated README accordingly to be vague.